tests/sync/test_settings.py

```diff
@@ -0,0 +1,36 @@
+import pathlib
+
+from kitovu.sync.settings import Settings, PluginSettings
+
+
+def test_load_a_sample_yaml_file():
+    settings = Settings.from_yaml_file(pathlib.PurePath('./tests/assets/smb_example_config.yml'))
```

Comment:
Could be a fixture? Maybe okay like this for a single test (though I like separating tests from setup/teardown as much as possible), definitely needed when there's going to be more.

Reply:
Well it is the method I try to test... I agree that making fixtures out of setup/teardown of objects, but if we move the tested method out of the test itself I think it gets a bit confusing